### PR TITLE
[el] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@petsoukos can you help us and review this automatic pull request, please ? thanks by advance

File: locales/el/el.json [hunk 1]
```diff
@@ -1,618 +1,618 @@
...
+    "Andorra": "Ανδόρας",
+    "Angola": "Αγκόλα",
+    "Anguilla": "Ανγκουίλλα",
+    "Another user has updated this resource since this page was loaded. Please refresh the page and try again.": "Ένας άλλος χρήστης έχει ενημερώσει αυτόν τον πόρο από τότε που φορτώθηκε αυτή η σελίδα. Παρακαλώ ανανεώστε τη σελίδα και προσπαθήστε ξανά.",
+    "Antarctica": "Ανταρκτική",
+    "Antigua And Barbuda": "Αντίγκουα και Μπαρμπούντα",
+    "API Token": "Διακριτικό API",
+    "API Token Permissions": "Άδειες συμβόλων API",
+    "API Tokens": "Μάρκες API",
+    "API tokens allow third-party services to authenticate with our application on your behalf.": "Τα API tokens επιτρέπουν στις υπηρεσίες τρίτων να πιστοποιούν με την εφαρμογή μας για λογαριασμό σας.",
```
Comment (petsoukos) [inline locales/el/el.json:45]: When we have to use untranslated English words or terms within a Greek sentence, we shouldn't write them in plural form. "API tokens" should be just "API token", although I cant find a source confirming this, I just know it from "English" and "Greek" school.

File: locales/el/el.json [hunk 2]
```diff
@@ -1,618 +1,618 @@
...
-    "British Indian Ocean Territory": "British Indian Ocean Territory",
-    "Browser Sessions": "Browser Sessions",
+    "Belarus": "Λευκορωσία",
+    "Belgium": "Βέλγιο",
+    "Belize": "Μπελίζ",
+    "Benin": "Μπενίν",
+    "Bermuda": "Βερμούδα",
+    "Bhutan": "Μπουτάν",
+    "Bolivia": "Βολιβία",
+    "Bonaire, Sint Eustatius and Saba": "Bonaire, Sint Eustatius και Sábado",
```
Comment (petsoukos) [inline locales/el/el.json:84]: [Μποναίρ, Άγιος Ευστάθιος και Σάμπα](https://el.wikipedia.org/wiki/%CE%9A%CE%B1%CF%81%CE%B1%CF%8A%CE%B2%CE%B9%CE%BA%CE%AE_%CE%9F%CE%BB%CE%BB%CE%B1%CE%BD%CE%B4%CE%AF%CE%B1)

File: locales/el/el.json [hunk 3]
```diff
@@ -1,618 +1,618 @@
...
-    "Choose Type": "Choose Type",
-    "Christmas Island": "Christmas Island",
+    "Bulgaria": "Βουλγαρία",
+    "Burkina Faso": "Μπουρκίνα Φάσο",
+    "Burundi": "Μπουρούντι",
+    "Cambodia": "Καμπότζη",
+    "Cameroon": "Καμερούν",
+    "Canada": "Καναδά",
+    "Cancel": "Ακύρωση",
+    "Cape Verde": "ακρωτήριο",
```
Comment (petsoukos) [inline locales/el/el.json:99]: Totally wrong
It is officially known as "[Δημοκρατία του Κάμπου Βέρντε](https://el.wikipedia.org/wiki/%CE%A0%CF%81%CE%AC%CF%83%CE%B9%CE%BD%CE%BF_%CE%91%CE%BA%CF%81%CF%89%CF%84%CE%AE%CF%81%CE%B9%CE%BF)"

File: locales/el/el.json [hunk 4]
```diff
@@ -1,618 +1,618 @@
...
+    "Card": "Κάρτα",
+    "Cayman Islands": "Νήσοι Κέιμαν",
+    "Central African Republic": "Κεντροαφρικανική Δημοκρατία",
+    "Chad": "Τσαντ",
+    "Changes": "Αλλαγή",
```
Comment (petsoukos) [inline locales/el/el.json:104]: "Αλλαγή" is change, changes is "Αλλαγές"

File: locales/el/el.json [hunk 5]
```diff
@@ -1,618 +1,618 @@
...
+    "Delete Selected": "Διαγραφή Επιλεγμένων",
+    "Delete Team": "Διαγραφή Ομάδας",
+    "Denmark": "Δανία",
+    "Detach": "Αποσυνδέσετε",
+    "Detach Resource": "Αποσύνδεση Πόρου",
+    "Detach Selected": "Αποσύνδεση Επιλεγμένων",
+    "Details": "Στοιχεία",
+    "Disable": "Απενεργοποιήσετε",
+    "Djibouti": "Τζιμπουτί",
+    "Do you really want to leave? You have unsaved changes.": "Θέλεις πραγματικά να φύγεις; Έχετε μη αποθηκευμένες αλλαγές.",
```
Comment (petsoukos) [inline locales/el/el.json:166]: First sentence addresses the user informally with the question, but then switches to a formal address stating the warning. If it makes sense. Both sentences should address the user using the same way.
Either: "Θέλεις πραγματικά να φύγεις; Έχεις μη αποθηκευμένες αλλαγές."
or: "Θέλετε πραγματικά να φύγετε; Έχετε μη αποθηκευμένες αλλαγές."

File: locales/el/el.json [hunk 6]
```diff
@@ -1,618 +1,618 @@
...
+    "Dominica": "Κυριακή",
```
Comment (petsoukos) [inline locales/el/el.json:167]: Ντομίνικα

File: locales/el/el.json [hunk 7]
```diff
@@ -1,618 +1,618 @@
...
+    "Dominican Republic": "Δομινικανή Δημοκρατία",
+    "Done.": "Γίνεται.",
```
Comment (petsoukos) [inline locales/el/el.json:169]: Ολοκληρώθηκε

**Comment by caouecs** [comment]
Can you do a merge pull request with your updates ?

File: locales/el/el.json [hunk 8]
```diff
@@ -1,618 +1,618 @@
...
-    "Editor users have the ability to read, create, and update.": "Editor users have the ability to read, create, and update.",
-    "Egypt": "Egypt",
-    "El Salvador": "El Salvador",
-    "Email": "Email",
-    "Email Address": "Email Address",
+    "Ecuador": "Εκουαδόρ",
+    "Edit": "Επεξεργασία",
+    "Edit :resource": "Επεξεργασία :resource",
+    "Edit Attached": "Επεξεργασία Συνημμένου",
+    "Editor": "Επεξεργασία",
```
Comment (petsoukos) [inline locales/el/el.json:176]: An editor in this case is referring to a person with the capacity to edit something within some organization (like a publisher, or auditor, etc.). When I first saw it I thought it meant something like a "WYSIWYG" editor, but reading the next string I realized it had a different meaning. So the Greek string should be "Συντάκτης" .

**Comment by petsoukos** [comment]
I keep finding issues. Most of it is OK but there are some things that are totally wrong and need correction before merging this.

**Comment by petsoukos** [comment]
> Can you do a merge pull request with your updates ?

Jeez, I just saw the branch name "Machine translation..." C'mon this needs a human's touch to be of good quality.

**Comment by caouecs** [comment]
We tried to help translators with an automatic translation, but, of course, it need a review by a human

**Comment by petsoukos** [comment]
> We tried to help translators with an automatic translation, but, of course, it need a review by a human

I will try in my spare time to finish reviewing it and try to fix the missed translations

**Comment by andrey-helldar** [comment]
@petsoukos, I have pushed the fixes. Can you check it again? Thank you.

**Review by petsoukos — APPROVED** [review]
The changes from my last comments are fixed.

**Comment by caouecs** [comment]
I merge ?

**Comment by petsoukos** [comment]
@caouecs It still has issues, I just haven't gotten around reviewing all translation strings.

**Comment by caouecs** [comment]
> @caouecs It still has issues, I just haven't gotten around reviewing all translation strings.

You can do update after ;)

**Pull request merged**